sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -1076,6 +1076,13 @@ public partial class ServerCertificateCustomValidationArgs
 }
 namespace Azure.Core.Serialization
 {
+    [System.AttributeUsageAttribute(System.AttributeTargets.Class)]
+    public sealed partial class DefaultSubClassAttribute : System.Attribute
+    {
+        public DefaultSubClassAttribute([System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)] System.Type type) { }
```

Comment:
To clarify - adding TrimmingAttributes.cs helps add this attribute here? And the guidance is avoiding differences between net5.0 and net6.0? It's fine that the others are different?

Reply:
@KrzysztofCwalina the public api surface still actually comes out different.  Not sure if this is just a side effect of our api export tool or a real difference.  Since you suggested we follow the NullableAttribute pattern can you take a look and see if this turned out the way you expected?